tests/integration/gitlab/test_generic_commands.py

```diff
@@ -23,6 +23,16 @@ def test_get_file_content(self):
             "https://github.com/packit-service/ogr\n\ntest1\ntest2\n"
         )
 
+    def test_get_file_content_resolve_dot(self):
+        file = self.project.get_file_content(
+            path="./README.md",
```

Comment:
I hope that we somehow don't end up with `././` :D

Reply:
good point, I changed it to use `normpath`